test/e2e/reconciler/namespace/controller_test.go

```diff
@@ -84,22 +84,15 @@ func TestNamespaceScheduler(t *testing.T) {
 				err = server.expect(namespace, unscheduledMatcher(nscontroller.NamespaceReasonUnschedulable))
 				require.NoError(t, err, "did not see namespace marked unschedulable")
 
-				t.Log("Creating a fake workload server")
-				logicalServer := framework.NewFakeWorkloadServer(t, server, server.orgClusterName)
-
-				t.Log("Create a cluster")
-				cluster1, err := framework.CreateWorkloadCluster(t, server.Artifact, server.kcpClient, logicalServer)
-				require.NoError(t, err, "failed to create cluster1")
-
-				framework.StartWorkspaceSyncer(t, ctx, sets.NewString(), cluster1, server, logicalServer)
-
-				err = server.expect(namespace, scheduledMatcher(cluster1.Name))
-				require.NoError(t, err, "did not see namespace marked scheduled for cluster1 %q", cluster1.Name)
+				syncerFixture := framework.NewSyncerFixture(t, sets.NewString(), server, server.orgClusterName, server.clusterName)
```

Comment:
Maybe add a comment here indicating why the syncer fixture is necessary (e.g. 'start a syncer against a workload cluster so that there is a ready cluster to schedule to')?